index.d.ts

```diff
@@ -1980,9 +1980,19 @@ export namespace stream {
 
 		class WorkbookReader extends Workbook {
 			constructor(input: string | import('stream').Stream, options: Partial<WorkbookStreamReaderOptions>);
+			input: import('stream').Stream;
+			options: WorkbookStreamReaderOptions;
+			stream: import('stream').Stream;
```

Comment:
I could not find this field in a workbook reader. Could I please add a permalink into the source code?